www/js/control/ExpandMenu.jsx

```diff
@@ -16,6 +16,8 @@ const ExpansionSection = (props) => {
       style={styles.section(colors.surface)}
       title={t(props.sectionTitle)}
       titleStyle={rowStyles.title}
+      accessible={true}
+      accessibilityLabel={t(props.sectionTitle) + ' spinbutton'}
```

Comment:
"spinbutton"? Are users going to know what that means?

Reply:
I used the React accessibilityRole documentation to come up with this (I hadn't heard of "spinbutton" either before reading this). It says:

> - **spinbutton** used to represent a button that opens a list of choices.

There are no other options I saw in this list that really matched the criteria better. I briefly spoke to a friend of mine who uses accessibility features and they said that most people they know who use screen reader have a pretty good idea of how web elements work purely by using the screenreader, so I wouldn't be surprised if spinbutton is a word that is commonly known among people who use screenreader. I'd be open to changing this if we find a better word or role to use.

https://reactnative.dev/docs/accessibility#accessibilityrole

Reply:
We can tweak further based on feedback from TCAT